test_harness.jl

```diff
@@ -7,6 +7,25 @@ kwargs = Kwargs.kwargs(; coverage=ENV["COVERAGE"],
                                      string("--compiled-modules=", ENV["COMPILED_MODULES"])],
                          test_args=ENV["TEST_ARGS"],
                          )
+commit_info = ""
+try
+    global commit_info
+    using LibGit2
+    repo = LibGit2.GitRepo(pwd())
+    commit = LibGit2.head(repo)
+    commit_sha = LibGit2.GitHash(commit)
+    commit_info = "commit $commit_sha of "
+catch e
+    @info "Could not determine commit of repo in current directory" exception=(e, catch_backtrace())
+end
+
+println("""
+###############
+# To reproduce this CI run locally, check out $(commit_info)this repo and
```

Comment:
```suggestion
# To reproduce this CI run locally, check out $(commit_info) on this repo and
```

Reply:
See `commit_info` above. It's setup to handle if we can't determine the commit hash.

Reply:
Aah, nice - missed that!